### PR TITLE
Update corefx to rc3-24210-10

### PR DESCRIPTION
**Pull request by cston**

### CODE REVIEW

**Comment by cston** [comment]
@tmat, @vsadov, @joshfree please review.
@mattgertz, @ManishJayaswal for approval.

**Comment by ManishJayaswal** [comment]
I asked Chuck to optimistically check this in without approval so that we can have a build that we can pass to next step in the chain. @MattGertz let us know if you disagree.

**Comment by joshfree** [comment]
changes LGTM

File: src/Debugging/Microsoft.DiaSymReader.PortablePdb/Microsoft.DiaSymReader.PortablePdb.nuspec [hunk 1]
```diff
@@ -13,22 +13,22 @@
...
-        <dependency id="System.Reflection" version="4.1.0-rc3-24208-04" />
-        <dependency id="System.Reflection.Metadata" version="1.3.0-rc3-24208-04" />
-        <dependency id="System.Reflection.Primitives" version="4.0.1-rc3-24208-04" />
-        <dependency id="System.Runtime" version="4.1.0-rc3-24208-04" />
-        <dependency id="System.Runtime.Extensions" version="4.1.0-rc3-24208-04" />
-        <dependency id="System.Runtime.InteropServices" version="4.1.0-rc3-24208-04" />
-        <dependency id="System.Text.Encoding" version="4.0.11-rc3-24208-04" />
-        <dependency id="System.Threading" version="4.0.11-rc3-24208-04" />
-        <dependency id="System.Xml.XDocument" version="4.0.11-rc3-24208-04" />
+        <dependency id="Microsoft.DiaSymReader" version="1.0.8-rc2-60610-01" />
```
Comment (tmat) [inline src/Debugging/Microsoft.DiaSymReader.PortablePdb/Microsoft.DiaSymReader.PortablePdb.nuspec]: The nuspec files don't contain version numbers anymore. Have you synced to the latest?

Comment (cston) [inline src/Debugging/Microsoft.DiaSymReader.PortablePdb/Microsoft.DiaSymReader.PortablePdb.nuspec]: This file was not updated to use `$coreFXVersionSuffix$` etc. I'll change it to use the variables in the next upgrade.

**Comment by tmat** [comment]
:+1:

**Pull request merged**